### ISSUE TITLE
NPE during scan with Wildfly 15

### ISSUE THREAD

**Issue by tillmann-schall** [classgraph/classgraph#320]
Hi,

when using classgraph 4.8.2 in combination with Wildfly 15 during scan the following exception occurs. The same problem already existed at least with 4.8.1

     [exec] 2019-02-19 07:55:02,986 [ServerService Thread] INFO  io.github.classgraph.ClassGraph                     - 2019-02-19T07:55:02.986+0100	ClassGraph	Found classpath element: D:\Development\JBossGroup\Wildfly\15.0.1\standalone\tmp\vfs\deployment\deploymenta0638eafce5c96a\webdav-servlet-2.0.5.jar-e87d18237c63c854\webdav-servlet-2.0.5.jar -> D:/Development/JBossGroup/Wildfly/15.0.1/standalone/tmp/vfs/deployment/deploymenta0638eafce5c96a/webdav-servlet-2.0.5.jar-e87d18237c63c854/webdav-servlet-2.0.5.jar
     [exec] 2019-02-19 07:55:02,987 [ServerService Thread] INFO  io.github.classgraph.ClassGraph                     - 2019-02-19T07:55:02.987+0100	ClassGraph	Exception in ClassLoaderHandler
     [exec] 2019-02-19 07:55:02,988 [ServerService Thread] INFO  io.github.classgraph.ClassGraph                     - 2019-02-19T07:55:02.987+0100	ClassGraph	java.lang.NullPointerException
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at nonapi.io.github.classgraph.utils.ReflectionUtils.invokeMethod(ReflectionUtils.java:287)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at nonapi.io.github.classgraph.classloaderhandler.JBossClassLoaderHandler.handleResourceLoader(JBossClassLoaderHandler.java:98)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at nonapi.io.github.classgraph.classloaderhandler.JBossClassLoaderHandler.handleRealModule(JBossClassLoaderHandler.java:175)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at nonapi.io.github.classgraph.classloaderhandler.JBossClassLoaderHandler.handle(JBossClassLoaderHandler.java:210)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at nonapi.io.github.classgraph.classpath.ClasspathFinder.<init>(ClasspathFinder.java:327)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at io.github.classgraph.Scanner.<init>(Scanner.java:148)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at io.github.classgraph.ClassGraph.scanAsync(ClassGraph.java:1079)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at io.github.classgraph.ClassGraph.scan(ClassGraph.java:1114)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at io.github.classgraph.ClassGraph.scan(ClassGraph.java:1151)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at io.github.classgraph.ClassGraph.scan(ClassGraph.java:1163)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at com.anaptecs.jeaf.fwk.core.impl.ReflectionToolsImpl.scanClassPath(ReflectionToolsImpl.java:271)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at com.anaptecs.jeaf.fwk.core.impl.ReflectionToolsImpl.getScanResult(ReflectionToolsImpl.java:238)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at com.anaptecs.jeaf.fwk.core.impl.ReflectionToolsImpl.getClassesWithAnnotation(ReflectionToolsImpl.java:213)
     [exec] 2019-02-19T07:55:02.987+0100	ClassGraph		at com.anaptecs.jeaf.fwk.core.MessageRepositoryImpl.loadResourcesFromClasspath(MessageRepositoryImpl.java:265)

Our scanner configuration is rather simple:
```
// Create new scanner for classpath
ClassGraph lScanner = new ClassGraph();
lScanner.enableAllInfo().enableClassInfo().enableRealtimeLogging();
// Execute scan
ScanResult lScanResult = `lScanner.scan();

```
To me it looks like some internal changes in Wildfly classloading have impact on classgraph.

**Comment by lukehutch** [comment]
@tillmann-schall Sorry, fixed and released as 4.8.3. Thanks for the report!

**Comment by tillmann-schall** [comment]
Thanks for fixing the NPE so fast
